routes/anonymousDeck.js

```diff
@@ -108,6 +98,90 @@ router.post("/create", async function (req, res) {
   }
 });
 
+router.post("/edit", async function (req, res) {
```

Comment:
this route seems to not be doing anything?

Reply:
This should be for editing the deck. You add/remove profiles, then press the checkmark icon, and it posts to this route.